Dockerfile

```diff
@@ -15,3 +15,5 @@ LABEL org.label-schema.build-date=$BUILD_DATE \
 RUN set -ex \
     && bin/elasticsearch-plugin install --batch repository-gcs \
     && bin/elasticsearch-plugin install --batch repository-s3
+
+RUN sed -r "s/filecount=[0-9]+,filesize=[0-9]+[kmg]/filecount=4,filesize=64m/" -i /usr/share/elasticsearch/config/jvm.options
```

Comment:
is this the same `sed` from our old image ? I am sure it will work, just wondering :)

Reply:
It hadn't even occurred to me to check the old image :man_facepalming:

I've updated it to use the same commands (realised I also missed the Java 8 GC log setting) as the old image, and tested it working.